src/App.js

```diff
@@ -54,6 +56,184 @@ const App = () => {
+              <p className="kaufman__table-text">Разбейте навык на элементы.</p>
+            </td>
+            <td>
+              <h4 className="kaufman__table-title">5</h4>
+              <p className="kaufman__table-text">
+                Приготовьте всё необходимое для занятий.
+              </p>
+            </td>
+          </tr>
+          <tr>
+            <td>
+              <h4 className="kaufman__table-title">6</h4>
+              <p className="kaufman__table-text">
+                Устраните препятствия для занятий.
+              </p>
+            </td>
+            <td>
+              <h4 className="kaufman__table-title">7</h4>
+              <p className="kaufman__table-text">
+                Выделите специальное время для занятий.
+              </p>
+            </td>
+            <td>
+              <h4 className="kaufman__table-title">8</h4>
+              <p className="kaufman__table-text">
+                Создайте быстрые петли обратной связи.
+              </p>
+            </td>
+            <td>
+              <h4 className="kaufman__table-title">9</h4>
+              <p className="kaufman__table-text">
+                Занимайтесь по расписанию, короткими интенсивными интервалами.
+              </p>
+            </td>
+            <td>
+              <h4 className="kaufman__table-title">10</h4>
+              <p className="kaufman__table-text">
+                Уделяйте внимание количеству и скорости.
+              </p>
+            </td>
+          </tr>
+        </table>
+        <img
+          className="kaufmanTriangle"
+          src={kaufmanTriangle}
+          height="877px"
+          width="877px"
+          alt="Blue Triangle"
+        />
+      </section>
+      <footer className="footer">
+        <div className="columns">
+          <div className="column__content">
+            <div className="content__copyright">
+              <img
+                className="footer__logo"
+                src={footerLogo}
+                width="228px"
+                height="32px"
+                alt="logo"
+              />
+              <p className="copyright__author">© Виктор Абанькин</p>
+            </div>
+            <div className="column__content-info">
+              <p className="content__column-heading">О практикуме</p>
+              <div className="column-links">
+                <a href="google.com" className="footer__links">
+                  Главная
+                </a>
+                <a href="google.com" className="footer__links">
+                  Концепция
+                </a>
+                <a href="google.com" className="footer__links">
+                  Наставники
+                </a>
+              </div>
+            </div>
+            <div className="column__content-social">
+              <p className="content__social-heading">Соцсети</p>
+              <div className="column-links">
+                <a href="facebook.com" className="footer__social-links">
```

Comment:
Try to click on this link and compare it with the 114th line, this one will not work. Why?

Reply:
fixed